kolibri/core/assets/src/views/CoreLogo/index.vue

```diff
@@ -1,5 +1,5 @@
 <template>
 
-  <img src="./kolibri-logo.svg">
+  <img src="./kolibri-logo.svg" alt="Kolibri logo">
```

Comment:
I think we should translate this, no?

Reply:
or maybe `alt=""` if we consider the logo to be decorative?

* https://www.w3.org/WAI/tutorials/images/decorative/
* https://webaim.org/techniques/alttext/#logos

Reply:
Thanks for fixing this, such a :woman_facepalming:, forgetting to localize strings in this PR...